WebDriverAgentLib/Categories/XCUIElement+FBUtilities.m

```diff
@@ -44,18 +46,19 @@
 
 @implementation XCUIElement (FBUtilities)
 
-- (id<FBXCElementSnapshot>)fb_takeSnapshot
+- (id<FBXCElementSnapshot>)fb_takeSnapshot:(BOOL)inDepth
 {
   NSError *error = nil;
-  self.fb_isResolvedFromCache = @(NO);
-  self.lastSnapshot = [self.fb_query fb_uniqueSnapshotWithError:&error];
+  self.lastSnapshot = inDepth
+    ? [self.fb_query fb_uniqueSnapshotWithError:&error]
+    : (id<FBXCElementSnapshot>)[self snapshotWithError:&error];
```

Comment:
I see, then the result would be the same as vanilla XCTest's `snapshotWithError`
(Just my reading note)